docs/REFERENCE.md

```diff
@@ -112,8 +112,8 @@ Err.of<TError>(value: TError): Err<TError>
 ```typescript
 import { Err } from 'type-safe-errors';
 
-class UserNotFound {
-  __brand!: "UserNotFound"
+class UserNotFound extends Error {
```

Comment:
what do you think about adding the `Error` suffix to example class names now that you extend from `Error`

Reply:
sounds good, I will update it.